topologies/6-22-stag-db-labels-2.json

```diff
@@ -0,0 +1,5963 @@
+{
+  "topology": {
+    "services": [
+      {
+        "serviceName": "service-0",
+        "instances": [
+          "service-0-instance-0",
```

Comment:
How are the instances being used here? And how does it help us with tests?

Reply:
I believe its used to mimic a service running on multiple instances. For each service call it randomly chooses an instance from this list for the span. I don't think this is specifically helping in any particular test but I kept it as is from the previous topology which was being used.